a worker --verbose calls run --verbose

dachary opened the pull request:
When a worker runs with --verbose, it also runs the jobs with --verbose.

Signed-off-by: Loic Dachary <loic@dachary.org>

ceph-jenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ceph.com//job/teuthology-pull-requests/1554/
Tests failed for this pull request.

ceph-jenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ceph.com//job/teuthology-pull-requests/1576/
Tests passed for this pull request.

zmc commented on teuthology/worker.py:
```diff
@@ -222,7 +222,7 @@ def run_with_watchdog(process, job_config):
 
 
-def run_job(job_config, teuth_bin_path):
+def run_job(job_config, teuth_bin_path, verbose):
```
If you don't provide a default value for the verbose arg, this will break:
https://github.com/dachary/teuthology/blob/wip-worker-verbose/teuthology/worker.py#L165

zmc commented on teuthology/test/test_worker.py:
```diff
@@ -73,7 +73,7 @@ def test_run_job_with_watchdog(self, m_tempfile, m_safe_dump, m_environ,
         m_popen.return_value = m_p
         m_t_config.results_server = True
-        worker.run_job(config, "teuth/bin/path")
+        worker.run_job(config, "teuth/bin/path", False)
```
If you end up adding a default value for ``verbose`` to ``run_job()``, I'd encourage either dropping these args or naming them, i.e. ``verbose=False``

dachary commented:
Thanks for catching this mistake. I introduced the problem while working on fixing flake8 messages. This should be ok now, and I added verbose=False as suggested.

ceph-jenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ceph.com//job/teuthology-pull-requests/1595/
Tests passed for this pull request.
